### DISCUSSION TITLE
記事を英語に翻訳し、公開してくれる機能がほしい

### DISCUSSION THREAD

**Discussion by YujiSoftware** [increments/qiita-discussions#884] [要望・アイデア・不具合]
Qiita には、有用な記事が多数あります。
しかし、それらが日本語であるために英語圏の方には届いていません。
技術記事は全世界共通で役立つことが多いのに、これはもったいないと思っております。

そこで、**[Cloud Translation API](https://cloud.google.com/translate/?hl=ja) などにより、記事を自動で英語に翻訳し、公開してくれる機能がほしいです。**

精度に関しては、例えば（英語→日本語の例ですが）[Google for Developer](https://developers.google.com/?hl=ja) が Cloud Translation API によって翻訳されていますが、違和感なく読むことができています。
そのため、機械翻訳した記事が検索のノイズになる（SEO 的に不利になる）ことはないかなと思います。

**Comment by tanaka-asami415** [comment]
@YujiSoftware 
リクエストありがとうございます。

> しかし、それらが日本語であるために英語圏の方には届いていません。
技術記事は全世界共通で役立つことが多いのに、これはもったいないと思っております。

上記について我々も同じ感覚を持っていますが、まだ具体的な施策は考えられていない状況です。
しばらく先になるとは思いますが、参考にさせていただきます🙇

**Comment by fygar256** [comment]
機械翻訳の精度がもう少し上がれば、あるいはもう充分かも知れませんが、そういう機能が欲しいです。
僕はネイティブな英語が不得手なので、Qiitaに搭載していただけると幸いです。
